src/l500/l500-private.h

```diff
@@ -560,5 +561,7 @@ namespace librealsense
         };
 
         class ac_trigger;
+
+        uint8_t weeks_since_factory_calibration(const std::string& serial);
```

Comment:
Please think how to separate this such that it can be used in a unit-test. Maybe check out what NIr just did with the max_usable_range feature.

Reply:
I don't understand. Separate from what?